apple/RNGestureHandlerEvents.h

```diff
@@ -24,12 +25,22 @@
                             withNumberOfTouches:(NSUInteger)numberOfTouches
                                    withDuration:(NSUInteger)duration
                                 withPointerType:(NSInteger)pointerType;
+#if !TARGET_OS_OSX && !TARGET_OS_TV
++ (RNGestureHandlerEventExtraData *)forPan:(CGPoint)position
+                      withAbsolutePosition:(CGPoint)absolutePosition
+                           withTranslation:(CGPoint)translation
+                              withVelocity:(CGPoint)velocity
+                       withNumberOfTouches:(NSUInteger)numberOfTouches
+                           withPointerType:(NSInteger)pointerType
+                            withStylusData:(NSDictionary *)stylusData;
+#else
```

Comment:
Why not keep the one with stylus data and pass `nil` when there's no data? Significantly reduces `#ifs`.

Reply:
I'm not sure if I understood correctly, but if you mean simply doing:  `withStylusData: nil;` it will crash the application.

Reply:
Yes, because it's not handled at all. But you could be checking if `stylusData` exists instead of checking the pressure.

Reply:
cc @m-bert

Reply:
Okay, I think I know what you mean. Let me know if [this change](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/16afa6eb59251f935fb65aa8d2f68762f663d2ca) is ok 😅